doc/changelogs/CHANGELOG_V16.md

```diff
@@ -0,0 +1,310 @@
+
+Contributed by James Snell - [#38112](https://github.com/nodejs/node/pull/38112)
+
+#### Toolchain and Compiler Upgrades
+
+* **build**: update Makefile to support fat binary (Ash Cripps) [#37861](https://github.com/nodejs/node/pull/37861)
+* **build**: warn for gcc versions earlier than 8.3.0 (Richard Lau) [#37935](https://github.com/nodejs/node/pull/37935)
+* **(SEMVER-MAJOR)** **doc**: update minimum supported Xcode to 11 (Michaël Zasso) [#37872](https://github.com/nodejs/node/pull/37872)
+* **(SEMVER-MAJOR)** **doc**: update minimum supported GCC to 8.3 (Michaël Zasso) [#37871](https://github.com/nodejs/node/pull/37871)
+* **(SEMVER-MAJOR)** **doc**: update AIX to GCC8 for v16.x (Ash Cripps) [#37677](https://github.com/nodejs/node/pull/37677)
+
+#### V8 9.0
+
+The V8 JavaScript engine is updated to V8 9.0, including performance tweaks and improvements.
+
+This update also brings the ECMAScript RegExp Match Indices, which provide the start and end indices of the captured string. The indices array is available via the `.indices` property on match objects when the regular expression has the `/d` flag.
+
+Contributed by Michaël Zasso - [#37587](https://github.com/nodejs/node/pull/37587)
+
+#### Other Notable Changes
+
+* **(SEMVER-MINOR)** **assert**: graduate assert.match and assert.doesNotMatch (James M Snell) [#38111](https://github.com/nodejs/node/pull/38111)
+* **(SEMVER-MAJOR)** **buffer**: expose btoa and atob as globals (James M Snell) [#37786](https://github.com/nodejs/node/pull/37786)
+* **(SEMVER-MINOR)** **http**: add http.ClientRequest.getRawHeaderNames() (simov) [#37660](https://github.com/nodejs/node/pull/37660)
+* **(SEMVER-MAJOR)** **lib,src**: update cluster to use Parent (Michael Dawson) [#36478](https://github.com/nodejs/node/pull/36478)
+* **(SEMVER-MINOR)** **module**: add support for `node:`‑prefixed `require(…)` calls (ExE Boss) [#37246](https://github.com/nodejs/node/pull/37246)
+* **(SEMVER-MINOR)** **repl**: add auto‑completion for `node:`‑prefixed `require(…)` calls (ExE Boss) [#37246](https://github.com/nodejs/node/pull/37246)
+
+### Semver-Major Commits
+
+* [[`324a6c235a`](https://github.com/nodejs/node/commit/324a6c235a)] - **(SEMVER-MAJOR)** **async_hooks**: add thisArg to AsyncResource.bind (James M Snell) [#36782](https://github.com/nodejs/node/pull/36782)
+* [[`d1e2184c8e`](https://github.com/nodejs/node/commit/d1e2184c8e)] - **(SEMVER-MAJOR)** **buffer**: expose btoa and atob as globals (James M Snell) [#37786](https://github.com/nodejs/node/pull/37786)
+* [[`1d8c022544`](https://github.com/nodejs/node/commit/1d8c022544)] - **(SEMVER-MAJOR)** **build**: update Makefile to support fat binary (Ash Cripps) [#37861](https://github.com/nodejs/node/pull/37861)
+* [[`a572a4e34e`](https://github.com/nodejs/node/commit/a572a4e34e)] - **(SEMVER-MAJOR)** **build**: reset embedder string to "-node.0" (Michaël Zasso) [#37587](https://github.com/nodejs/node/pull/37587)
```

Comment:
@targos, do you think it makes sense to remove `semver-major` indicator from some of the commits in #37587 such as this one? And potentially do the same for commits related to #37330 too?

Reply:
Yes, I'll prepare a patch for the changelog once you've finalized the commit list. Ping me again when it's ready, please.

Reply:
@targos, I know it's rather last minute but the [changelog](https://github.com/nodejs/node/blob/fc6a49f8dff2e2261fdf0d461abf7d25337b6614/doc/changelogs/CHANGELOG_V16.md) commit list is finalised - I don't expect to make any further updates.

Reply:
@BethGriggs I pushed the change to this branch

Reply:
@targos, thank you - they major list looks much less noisy now. I'll squash it down shortly with the other proposed changes.